include/fmt/format.h

```diff
@@ -1568,9 +1574,9 @@ template <typename OutputIt, typename Char, typename UInt> struct int_writer {
     if (group == groups.cend()) size += sep_size * ((n - 1) / groups.back());
     char digits[40];
     format_decimal(digits, abs_value, num_digits);
-    basic_memory_buffer<Char> buffer;
+    // TODO(kammce): remove the * 2!
+    std::array<Char, inline_buffer_size * 2> buffer;
```

Comment:
This won't work because there is no limit on the separator size. What you could do instead is pass an allocator to memory buffer that throws/aborts whenever it wants to allocate.

Reply:
I'll make the change for this this weekend.